### PR TITLE
Update standards.rst

### PR DESCRIPTION
**Pull request by Coffee2CodeNL**
Yoda Conditions decrease readability and most IDEs have inspections to prevent Assignment in Conditions, use this instead.

### CODE REVIEW

**Review by ghost — CHANGES_REQUESTED** [review]
There is a spelling mistake in your change: `[...] make sure it hilights it [...]`. Would you like to correct it?

**Comment by iSDP** [comment]
@ricknox fixed =] thanks!

**Review by ghost — APPROVED** [review]
√

**Review by Majkl578 — APPROVED** [review]

**Review by greg0ire — APPROVED** [review]

File: contributing/code/standards.rst [hunk 1]
```diff
@@ -147,9 +147,7 @@ Structure
 
 * Always use `identical comparison`_ unless you need type juggling;
 
-* Use `Yoda conditions`_ when checking a variable against an expression to avoid
-  an accidental assignment inside the condition statement (this applies to ``==``,
-  ``!=``, ``===``, and ``!==``);
+* Use your IDE to check for Assignment In Conditions and make sure it highlights it as an error.;
```
Comment (sstok) [inline contributing/code/standards.rst]: No need for a dot _and_ semicolon, a `;` in a sentence indicates a break but the subject is not stopped as done with a period. Using only `;` is enough.

**Comment by greg0ire** [comment]
Please format your commit message according to these rules : 

- The commit message subject must be less than 50 characters and tell people *what* you did.
- The commit message body should tell people *why* you did it. It is optional but highly recommended.

Bad example :

```
Fixed bug #989 by removing call to defraculate()
```

Good example:

```
Remove call to defraculate()

Calling this function caused a bug because it interferes 
with calls to getSchmeckles().
Fixes #989
```

`Update file.ext` is github's default and is terrible, don't be afraid to change it.

**Comment by iSDP** [comment]
Closing this one, too much wrong now.
Opening a new one soon.

**Comment by iSDP** [comment]
New PR: https://github.com/symfony/symfony-docs/pull/9201

File: doctrine/reverse_engineering.rst [hunk 2]
```diff
@@ -4,6 +4,14 @@
 How to Generate Entities from an Existing Database
 ==================================================
 
+.. caution::
```
Comment (ricknox) [inline doctrine/reverse_engineering.rst]: I guess that this was the wrong branch when you did a commit?

Comment (iSDP) [reply doctrine/reverse_engineering.rst]: I have no clue what happened haha, that's why I made a new PR.

**Pull request closed without merging**